tests/test_basic_functionality.py

```diff
@@ -251,9 +254,7 @@ def test_tree_coerce_dict() -> None:
 
 
 def test_tree_coerce_str() -> None:
-    d = """"""
     s = "test"
-    assert LayeredConfigTree._coerce(d, s) == (None, s)
```

Comment:
Why are these lines removed?

Reply:
I think that we can't have None for data. Perhaps I'm wrong, though, in which case the typing is messed up here

Reply:
Don't forget to re-remove these lines.